src/main/clojure/jarl/builtins/time.clj

```diff
@@ -1,23 +1,102 @@
 (ns jarl.builtins.time
   (:require [jarl.builtins.utils :refer [check-args]]
             [jarl.exceptions :as errors]
-            [jarl.utils :refer [instant-to-ns]])
-  (:import (java.time Instant ZoneId LocalDateTime ZoneOffset ZonedDateTime)
+            [jarl.utils :refer [instant-to-ns ns-to-instant]]
+            [clojure.string :as str])
+  (:import (java.time ZoneId LocalDateTime ZoneOffset ZonedDateTime Duration)
            (java.time.format TextStyle DateTimeFormatter)
-           (java.util Locale)))
+           (java.util Locale)
+           (java.time.temporal ChronoUnit)))
 
-(defn parse-iso-datetime [s]
-  (-> s
-      (LocalDateTime/parse DateTimeFormatter/ISO_LOCAL_DATE_TIME)
-      (.toInstant ZoneOffset/UTC)))
+; Courtesy of https://github.com/newm4n/go-dfe
+(def go->java-map
```

Comment:
Can be made private?

Reply:
Sure! I'll have that included with the next PR, which is likely to be submitted this evening :)